packages/picasso-charts/src/BarChart/BarChart.tsx

```diff
@@ -64,6 +64,10 @@ export interface Props<K extends string | number | symbol>
   showBarLabel?: boolean
   /** If set false, animation of bar will be disabled */
   isAnimationActive?: boolean
+  /** Makes X-axis show only every Nth tick. `0` hides all ticks, `1` shows all ticks (default behavior) */
+  showEveryNthTickOnXAxis?: 0 | 1 | 2 | 3
+  /** Makes Y-axis show only every Nth tick. `0` hides all ticks, `1` shows all ticks (default behavior) */
+  showEveryNthTickOnYAxis?: 0 | 1 | 2 | 3
```

Comment:
Wdyt about calling this prop the same name as recharts?

either having a single prop:
```suggestion
  interval?: {
    x: 0 | 1 | 2 | 3,
    y: 0 | 1 | 2 | 3,
  }
```

or having a separate prop for x and y:
```suggestion
  xInterval?: 0 | 1 | 2 | 3
  /** Makes Y-axis show only every Nth tick. `0` hides all ticks, `1` shows all ticks (default behavior) */
  yInterval?: 0 | 1 | 2 | 3
```

Reply:
@augustobmoura The `interval` property seems a bit unclear, I think, so it is better to keep it verbose enough so users can see what it does right from the name itself